rllib/utils/replay_buffers/prioritized_episode_replay_buffer.py

```diff
@@ -16,6 +16,104 @@
 
 
 class PrioritizedEpisodeReplayBuffer(EpisodeReplayBuffer):
+    """Prioritized Replay Buffer that stores episodes by their ID.
+
+    This replay buffer stores episode data (more specifically `EpisodeV2`
+    objects) and implements prioritized experience replay first proposed
+    in the paper by Schaul et al. (2016, https://arxiv.org/abs/1511.05952).
+
+    Implementation is based on segment trees as suggested by the authors of
+    the cited paper, i.e. we use proportional prioritization with an order
+    of O(log N) in updating and sampling.
+
+    Each "row" (a slot in a deque) in the buffer is occupied by one episode. If an
+    incomplete episode is added to the buffer and then another chunk of that episode is
+    added at a later time, the buffer will automatically concatenate the new fragment to
+    the original episode. This way, episodes can be completed via subsequent `add`
+    calls.
+
+    Sampling returns batches of size B (number of 'rows'), where each row is a tuple
+    of the form
+
+    `(o_t, a_t, sum(r_t+1:t+n), o_t+n)`
+
+    where `o_t` is the observation in `t`, `a_t` the action chosen at observation `o_t`,
+    `o_t+n` is the observation `n` timesteps later and `sum(r_t+1:t+n)` is the sum of
+    all rewards collected over the time steps between `t+1` and `t+n`. The `n`-step can
+    be chosen freely when sampling and defaults to `1`. If `n_step` is a tuple it is
+    sampled uniformly across the interval defined by the tuple (for each row in the
+    batch).
+
+    Each batch contains in addition to the data tuples presented above two further
```

Comment:
nit: "Each batch contains - in addition to the data tuples presented above - two ..."